Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -225,6 +229,48 @@ public ImageDimensions GetImageSize(string path)
             }
         }
 
+        /// <inheritdoc />
+        /// <exception cref="ArgumentNullException">The path is null.</exception>
+        /// <exception cref="FileNotFoundException">The path is not valid.</exception>
+        /// <exception cref="SkiaCodecException">The file at the specified path could not be used to generate a codec.</exception>
+        [SuppressMessage("Microsoft.Performance", "CA1814:PreferJaggedArraysOverMultidimensional")]
+        public string GetImageHash(string path)
+        {
+            if (path == null)
+            {
+                throw new ArgumentNullException(nameof(path));
+            }
+
+            if (!File.Exists(path))
+            {
+                throw new FileNotFoundException("File not found", path);
+            }
+
+            using (var bitmap = GetBitmap(path, false, false, null))
```

Comment:
This obviously fails to run on remote images, what should we do about it?

Reply:
This should be called under `if (info.IsLocalFile)` condition, so it shouldn't be a problem now.